include/pagmo/algorithms/cmaes.hpp

```diff
@@ -83,8 +83,23 @@ class cmaes
 {
 public:
     /// Single entry of the log (gen, fevals, best, dx, df, sigma)
-    typedef std::tuple<unsigned int, unsigned long long, double, double, double, double> log_line_type;
-    /// The log
+    /// Single data line for the algorithm's log.
+    /**
+     * A log data line is a tuple consisting of:
+     * - the generation number,
+     * - the number of function evaluations
+     * - the best fitness vector so far,
+     * - the population flatness evaluated as the distance between the decisions vector of the best and of the worst
+     * individual,
+     * - the population flatness evaluated as the distance between the fitness of the best and of the worst individual.
+     */ typedef std::tuple<unsigned int, unsigned long long, double, double, double, double>
```

Comment:
This should probably go onto the next line. Kinda surprised clang-format did not pick it up.